File: Beecrowd/2339 Aviões de Papel/main.cpp
#include <iostream>

using namespace std;

int main(){
    int c, p, f, res;

    cin >> c >> p >> f;

    res = c * f;

    if(res <= p)
        cout << "S" << endl;
    else
        cout << "N" << endl;

    return 0;
}

File: Beecrowd/2339 Aviões de Papel/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int c, p, f, res;

        c = sc.nextInt();
        p = sc.nextInt();
        f = sc.nextInt();

        res = c * f;

        if (res <= p)
            System.out.println("S");
        else
            System.out.println("N");

        sc.close();
    }

}
